### PR TITLE
Create codeql.yml

### PR DESCRIPTION
**Pull request by jukzi**

### CODE REVIEW

**Comment by jukzi** [comment]
output: https://github.com/eclipse-pde/eclipse.pde/security/code-scanning?query=is%3Aopen+pr%3A698

**Comment by szarnekow** [comment]
> output: https://github.com/eclipse-pde/eclipse.pde/security/code-scanning?query=is%3Aopen+pr%3A698

For me this is a 404

**Review by HannesWell — COMMENTED** [review]
Looks good, but I wonder how committer get notified if new issues aris?
Or do comitter get a notification on PRs in that case?

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: CodeQL call
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '45 8 * * 1'
```
[the remark this answers is not in the document]
Comment (jukzi) [reply .github/workflows/codeql.yml:9]: yes. the result of master branch is shown in the security tab, while for PRs it has to be queried with a search. The results
are only visible for committers @szarnekow 
notifications could be configured in the project settings, but by now i would be more interested to get away from ancient technical debt

**Pull request merged**